shared/chat/selectable-big-team-channel-container.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import * as Constants from '../constants/chat2'
+import SelectableBigTeamChannel from './selectable-big-team-channel'
+import * as Types from '../constants/types/chat2'
+import {namedConnect} from '../util/container'
+
+type OwnProps = {|
+  conversationIDKey: Types.ConversationIDKey,
+  isSelected: boolean,
+  onSelectConversation: () => void,
+|}
+
+const mapStateToProps = (state, {conversationIDKey}) => ({
+  _meta: Constants.getMeta(state, conversationIDKey),
```

Comment:
maybe better to extract channelname and teamname here so if other meta changes it won't rerender (i know this is just moved around)